lib/usdUfe/utils/editRouterContext.h

```diff
@@ -33,12 +33,18 @@ namespace USDUFE_NS_DEF {
 
 class USDUFE_PUBLIC StackedEditRouterContext : public PXR_NS::TfStacked<StackedEditRouterContext>
 {
-protected:
+public:
     /*! \brief Retrieve the current targeted layer.
      * \return The targeted layer. Null if the edit target was not changed.
      */
     const PXR_NS::SdfLayerHandle& getLayer() const;
 
+    /*! \brief Retrieve the routed stage.
+     * \return The stage that is being routed. Null if the edit target was not changed.
```

Comment:
Its not really true that it returns "Null". Doesn't this one return an empty stage? And the one above an empty layer?

Reply:
No, it does return a null pointer if the stage edit target was not changed.

Reply:
Iyt may not be obvious from teh code: if this edit router set a edit target, its _stage won't be null, if it did not, it asks its "parent" edit router in teh stack and if all teh chain all teh way to the top did not set an edit target, it will end-up returning null.

Reply:
Oh sorry I was confused by this:
```
static const PXR_NS::UsdStagePtr empty;
return empty;
```
I missed the "Ptr" part of it. Do you need this? Why can't you just return nullptr or {}

Reply:
Oh it was because the pointer is returned by reference. I can change that.

Reply:
The getLayer method above is the same const - can it be changed?